src/Arcus.Templates.ServiceBus.Topic/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM mcr.microsoft.com/dotnet/sdk:6.0.301-alpine3.14 AS base
+FROM mcr.microsoft.com/dotnet/sdk:8.0-alpine AS base
```

Comment:
The runtime base image that must be used, should not be an sdk version.
We should use one of [these](https://hub.docker.com/_/microsoft-dotnet-runtime/).


I would suggest mcr.microsoft.com/dotnet/runtime:8.0-alpine

Using an SDK image as the base-image for the runtime version of the container is too bloated.  It contains all kind of stuff that you don't need during runtime.  

See also https://github.com/arcus-azure/arcus.templates/issues/854

Reply:
Will be looked over in a follow-up PR.